macros/GroupData.json

```diff
@@ -208,26 +209,22 @@
                             "CSSStyleSheet",
                             "CSSSupportsRule",
                             "LinkStyle",
+                            "MediaQueryList",
+                            "MediaQueryListEvent",
```

Comment:
I kept [`MediaQueryListEvent`](https://developer.mozilla.org/docs/Web/API/MediaQueryListEvent) as it’s part of “CSSOM View”.